src/server/handlers/validator.rs

```diff
@@ -0,0 +1,59 @@
+//! Expose the subnet actor validator set
+
+use crate::config::ReloadableConfig;
+use crate::lotus::client::LotusJsonRPCClient;
+use crate::lotus::LotusClient;
+use crate::server::JsonRPCRequestHandler;
+use anyhow::anyhow;
+use async_trait::async_trait;
+use cid::Cid;
+use ipc_subnet_actor::Validator;
+use serde::{Deserialize, Serialize};
+use std::str::FromStr;
+use std::sync::Arc;
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct QueryValidatorSetParams {
+    pub subnet: String,
+    pub tip_set: String,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct QueryValidatorSetResponse {
+    /// The address of the created subnet
+    pub validator_set: Vec<Validator>,
```

Comment:
I will tag you on a PR that I am working on as this may change slightly.

Reply:
np np, will keep an eye for this